src/sentry/static/sentry/app/components/errorBoundary.tsx

```diff
@@ -70,13 +75,14 @@ class ErrorBoundary extends React.Component<Props, State> {
   }
 
   componentWillUnmount() {
+    this._isMounted = false;
     if (this.unlistenBrowserHistory) {
       this.unlistenBrowserHistory();
     }
   }
 
-  // XXX: browserHistory.listen does not have a correct return type.
-  unlistenBrowserHistory: any;
+  private unlistenBrowserHistory?: ReturnType<typeof browserHistory.listen>;
```

Comment:
Thank you for typing this 😍

Reply:
I use it on oddball properties/methods that should really stay inside a component. 

IIRC it doesn't stop someone from passing it to a child component but it's an additional hint that they shouldn't do it.